Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -410,4 +421,19 @@ namespace EMStudio
         m_renderOverlay.setGeometry(geometry());
         m_viewportUiManager.Update();
     }
+
+    void AnimViewportWidget::OnBeginPrepareRender()
+    {
+        if (m_debugDisplay)
+        {
```

Comment:
nit: We could early return here if `m_debugDisplay` is nullptr to avoid the indentation (or if this _should_ never happen we could add an Assert too?)

Reply:
I don't think it makes much difference here because it's such a short function, but will bear in mind for the future.